gemrb/GUIScripts/GUISTORE.py

```diff
@@ -54,14 +53,13 @@
 inventory_slots = ()
 total_price = 0
 total_income = 0
-if GameCheck.IsIWD2():
-	ItemButtonCount = 6
-else:
-	ItemButtonCount = 4
-RepModTable = None
-SpellTable = None
 BarteringPC = 0
 
+ItemButtonCount = 4 if GameCheck.IsIWD2() else 6
+CureTable = GemRB.LoadTable("speldesc", True) # additional info not supported by core (PST)
+RepModTable = GemRB.LoadTable ("repmodst", True) # all but PST; FIXME: does IWD2 sufffer a 10x price increase?
```

Comment:
well yes, that's why it was ifdefed

Reply:
right, but why is the data seemingly wrong? IESDP makes it seem like IWD2 data behaves identically to the others. If we really want to ignore the file seems like we ought to override it with an empty one, at least then it is mod-able.

Reply:
like the commit says, it's a leftover from the previous game they started with (bg2). IWDs don't have any reputation system at all. But yeah, a special 2da override would also work, though since it has to be an override, it's not easily moddable except manually.